src/VSCodeUI.ts

```diff
@@ -53,6 +54,32 @@ export namespace VSCodeUI {
         }
     }
 
+    export function setJavaHomeIfRequired(terminal: Terminal): void {
+        const javaHome: string = workspace.getConfiguration("java").get<string>("home");
+        const setJavaHome: boolean = workspace.getConfiguration("maven").get<boolean>("set.javaHome");
+        if (setJavaHome && javaHome) {
```

Comment:
What if the user needs to specify `JAVA_HOME` by his/her own without installing the redhat java extension?
So is it good to allow users to do that by 2 configurations as following: 
```
          "maven.javaHome.from": {
            "type":"string",
            "default": "system envrionment",
            "enum": [
              "system envrionment",
              "extension: redhat.java",
              "specified in maven.javaHome.path"
            ]
          },
          "maven.javaHome.path": {
            "type" :"string",
            "default": ""
          }
```
Or is it too complicated/noisy to have so many configuration entries?